lib/identity_doc_auth/acuant/request.rb

```diff
@@ -100,7 +100,7 @@ def handle_invalid_response(http_response)
         config.exception_notifier&.call(exception)
         IdentityDocAuth::Response.new(
           success: false,
-          errors: { network: I18n.t('errors.doc_auth.acuant_network_error') },
```

Comment:
Embarrassingly, I left in a reference to global `I18n` when I moved this over. Unfortunately, ActiveSupport has an i18n dependency, so it "covered" for this